spec/support/platform_helpers.rb

```diff
@@ -67,6 +67,13 @@ def windows_2008r2_or_later?
   components.length >= 2 && components[0] >= 6 && components[1] >= 1
 end
 
+def windows_2012r2?
+  return false unless windows?
+  wmi = WmiLite::Wmi.new
+  host = wmi.first_of("Win32_OperatingSystem")
+  (host["version"] && host["version"].start_with?("6.3"))
```

Comment:
There are three methods here now checking `host["version"]`, please create another method that uses WMI and returns `host["version"]`, and update these three methods to use it to avoid the duplicated pattern.